### PR TITLE
[DT-90] new CDR with AI/AN for preprod

### PR DESCRIPTION
**Pull request by chenchals**
Description:
---
PR for out-of-band push of preprod CDR config changes 
- @tarekmamdouh to confirm that wgs/microarray properties are correct in config changes "cdrVersion": 11 node
- @dmohs FYI
- Before running out-of-band I will be execute publish commands after getting security approval:
- then run update-cdr-config command

---
**PR checklist**

- [ ] This PR meets the Acceptance Criteria in the JIRA story
- [ ] The JIRA story has been moved to Dev Review
- [ ] This PR includes appropriate unit tests
- [ ] I have added explanatory comments where the logic is not obvious
- [ ] I have run and tested this change locally, and my testing process is described here
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later
- [ ] If this includes an API change, I have run the E2E tests on this change against my local server with [yarn test-local](https://github.com/all-of-us/workbench/blob/master/e2e/README.md#examples) because this PR won't be covered by the CircleCI tests 
- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer in [Slack](https://pmi-engteam.slack.com/archives/C02MWP2RN5P)
- [ ] If this change impacts deployment safety (e.g. removing/altering APIs which are in use) I have documented these in the description
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and updated the appropriate API consumers
  * AoU UI
  * [Perf tests](https://github.com/broadinstitute/mcnulty/blob/develop/src/test/scala/services/AoU.scala)
  * [Researcher Directory export](https://github.com/all-of-us/workbench/wiki/Researcher-Directory-(RDR-export))
  * Cron tasks - for Offline*Controllers
  * SumoLogic - for EgressAlert

### CODE REVIEW

File: api/config/cdr_config_preprod.json [hunk 1]
```diff
@@ -152,6 +150,45 @@
...
       "microarrayVcfSingleSampleStoragePath": "microarray/vcf/single_sample/alpha3"
+    },
+    {
+      "cdrVersionId": 10,
+      "name": "All of Us Registered Tier Dataset v7",
+      "isDefault": true,
+      "bigqueryProject": "fc-aou-cdr-preprod",
+      "bigqueryDataset": "R2022Q4R3",
+      "creationTime": "2022-11-15 00:00:00Z",
+      "releaseNumber": 10,
```
Comment (tarekmamdouh) [inline api/config/cdr_config_preprod.json:161]: I don't know what uses the releaseNumber, but seems it has to be incremented

Comment (chenchals) [reply api/config/cdr_config_preprod.json:161]: good catch!

**Comment by tarekmamdouh** [comment]
I think storageBasePath will have to updated later once hail and plink files are created and available under v7 directory. This is still in progress. I'll create a task to update this property.
I am actually suspecting that it's currently incorrect, because it has to be v6 instead of 5.

**Review by tarekmamdouh — APPROVED** [review]
LGTM at the moment, will need another update once plink and hail files are available.

**Review by dolbeew — APPROVED** [review]

**Pull request merged**